tests/puppeteer/campaigns/functional/BO/design/pages/03_categoriesPagesBulkAction.js

```diff
@@ -0,0 +1,289 @@
+      await expect(disableTextResult).to.be.equal(this.pageObjects.pagesPage.successfulUpdateStatusMessage);
+      const numberOfPagesInGrid = await this.pageObjects.pagesPage.getNumberFromText(
+        this.pageObjects.pagesPage.pageGridTitle);
+      await expect(numberOfPagesInGrid).to.be.at.most(numberOfPages);
+      /* eslint-disable no-await-in-loop */
+      for (let i = 1; i <= numberOfPagesInGrid; i++) {
+        const textColumn = await this.pageObjects.pagesPage.getTextContent(
+          this.pageObjects.pagesPage.pagesListTableColumn.replace('%ROW', i).replace('%COLUMN', 'active'),
+        );
+        await expect(textColumn).to.contains('clear');
+      }
+      /* eslint-enable no-await-in-loop */
+    });
+    it('should enable pages with Bulk Actions and check Result', async function () {
+      const enableTextResult = await this.pageObjects.pagesPage.changePagesEnabledColumnBulkActions(true);
+      await expect(enableTextResult).to.be.equal(this.pageObjects.pagesPage.successfulUpdateStatusMessage);
+      const numberOfPagesInGrid = await this.pageObjects.pagesPage.getNumberFromText(
+        this.pageObjects.pagesPage.pageGridTitle,
+      );
+      await expect(numberOfPagesInGrid).to.be.at.most(numberOfPages);
+      /* eslint-disable no-await-in-loop */
+      for (let i = 1; i <= numberOfPagesInGrid; i++) {
+        const textColumn = await this.pageObjects.pagesPage.getTextContent(
+          this.pageObjects.pagesPage.pagesListTableColumn.replace('%ROW', i).replace('%COLUMN', 'active'),
+        );
+        await expect(textColumn).to.contains('check');
+      }
+      /* eslint-enable no-await-in-loop */
+    });
+  });
+  // 3 : Delete Pages created with bulk actions
+  describe('Delete pages with Bulk Actions', async () => {
+    it('should filter list by Title', async function () {
+      await this.pageObjects.pagesPage.filterPages(
+        'input',
+        'meta_title',
+        'todelete',
+      );
+      const textResult = await this.pageObjects.pagesPage.getTextContent(
+        this.pageObjects.pagesPage.pagesListTableColumn.replace('%ROW', '1').replace('%COLUMN', 'meta_title',
+        ),
+      );
+      await expect(textResult).to.contains('todelete');
+    });
+    it('should delete pages with Bulk Actions and check Result', async function () {
+      const deleteTextResult = await this.pageObjects.pagesPage.deletePagesBulkActions();
+      await expect(deleteTextResult).to.be.equal(this.pageObjects.pagesPage.successfulMultiDeleteMessage);
+    });
+    it('should reset all filters', async function () {
+      await this.pageObjects.pagesPage.resetFilter(this.pageObjects.pagesPage.pagefilterResetButton);
+      const numberOfPagesAfterFilter = await this.pageObjects.pagesPage.getNumberFromText(
+        this.pageObjects.pagesPage.pageGridTitle);
+      await expect(numberOfPagesAfterFilter).to.be.equal(numberOfPages);
+    });
+  });
+});
```

Comment:
There are many duplicate in this file. It should be interesting to create dedicated methods.